app/controllers/nonsipp/membertransferout/TransfersOutCYAController.scala

```diff
@@ -123,32 +163,72 @@ object TransfersOutCYAController {
     memberName: String,
     index: Max300,
     journeys: List[TransfersOutCYA],
-    mode: Mode
+    mode: Mode,
+    viewOnlyUpdated: Boolean,
+    optYear: Option[String] = None,
+    optCurrentVersion: Option[Int] = None,
+    optPreviousVersion: Option[Int] = None,
+    compilationOrSubmissionDate: Option[LocalDateTime] = None
   ): FormPageViewModel[CheckYourAnswersViewModel] = {
-    val heading: InlineMessage = mode.fold(
-      normal = "checkYourAnswers.heading",
-      check = Message("transfersOutCYAController.heading.check", memberName)
-    )
+    val heading: InlineMessage =
+      mode
+        .fold(
+          normal = "checkYourAnswers.heading",
+          check = Message("transfersOutCYAController.heading.check", memberName),
+          viewOnly = "transfersOutCYAController.viewOnly.title"
+        )
 
     FormPageViewModel[CheckYourAnswersViewModel](
-      title = "checkYourAnswers.title",
+      mode = mode,
+      title = mode
+        .fold(
+          normal = "checkYourAnswers.title",
+          "",
```

Comment:
I think this may need to be "Change details of transfers out for this member" based on the content document attached to PSR-525. May need to be added to messages.en.

Reply:
Hi @kberec , it's working fine without adding the check.

Reply:
Sorry, now it's added in title.